awx/main/validators.py

```diff
@@ -48,24 +48,32 @@ def validate_pem(data, min_keys=0, max_keys=None, min_certs=0, max_certs=None):
 
     # Build regular expressions for matching each object in the PEM file.
     pem_obj_re = re.compile(
-        r'^(-{4,}) *BEGIN ([A-Z ]+?) *\1[\r\n]+' +
-        r'(.+?)[\r\n]+\1 *END \2 *\1[\r\n]?(.*?)$', re.DOTALL,
+        r'^-{5}\ *BEGIN ([A-Z ]+?)\ *-{5}' +
```

Comment:
I'm also having trouble understanding what the ultimate meaning of the `\` is after `{5}` here. Removing that character doesn't seem to affect the parsing of the expression. Perhaps it's a stray artifact from the old style of this regex?

Reply:
I put those there to mark zero or many whitepsaces `\ *` but I took it them out again.